fda/registration.py

```diff
@@ -144,8 +166,8 @@ def shift_registration(fd, maxiter=5, tol=1e-2, ext="default", alpha=1,
     max_diff = tol + 1
     iter = 0
 
-    # Saves the array in all the interval if it will be adjusted to slice
-    if not periodic:
+    # Auxiliar array if the domain will be restricted
+    if extrapolation == ExtrapolationType.slice:
```

Comment:
Comparison with singletons should be done using the `is` identity operator.

Reply:
Fixed in 9c17e4d.